fastlane/Fastfile

```diff
@@ -95,11 +96,17 @@ desc "Creates GitHub release and publishes react-native-purchases and react-nati
 lane :release do |options|
   version_number = current_version_number
   is_prerelease = Gem::Version.new(version_number).prerelease?
+  is_hotfix = !is_prerelease ? hotfix?(package_name) : false
```

Comment:
Note that I'm only checking the version for the base package, not for the paywalls package. So this assumes they will always have the same version and be deployed one after another.